.github/workflows/codeql.yml

```
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
```

Should also include release branches:

```suggestion
    branches:
      - 'master'
      - 'releases/v*'
```